go.mod

```diff
@@ -34,19 +34,19 @@ require (
 	github.com/Azure/azure-sdk-for-go v57.1.0+incompatible
 	github.com/Azure/azure-service-bus-go v0.11.1
 	github.com/Azure/azure-storage-queue-go v0.0.0-20191125232315-636801874cdd
-	github.com/Azure/go-autorest/autorest v0.11.20
-	github.com/Azure/go-autorest/autorest/adal v0.9.13
+	github.com/Azure/go-autorest/autorest v0.11.21
+	github.com/Azure/go-autorest/autorest/adal v0.9.16
```

Comment:
Are those unrelated version bumps needed?
Can we push them separately?

Reply:
no, they are not needed.

I wanted to also remove transitive dependencies to clean `go.sum`, but it is tougher than expected.
I'll remove them

Reply:
the rabbit hole is here at the moment

https://github.com/etcd-io/etcd/issues/13390